lib/readwise/client.rb

```diff
@@ -2,6 +2,7 @@
 require 'net/http'
 require_relative 'book.rb'
 require_relative 'highlight.rb'
+require_relative 'tag.rb'
```

Comment:
The `.rb` isn't needed here, I opened https://github.com/andjosh/readwise-ruby/pull/7 to remove the others.